### ISSUE TITLE
Can you insert records by external ID using this test data factory

### ISSUE THREAD

**Issue by kensation** [benahm/TDF#19]
Quick question - Can you insert records by external ID using this test data factory?

**Comment by conrjac** [comment]
Yes, see: https://github.com/benahm/TestDataFactory/blob/master/docs/EXAMPLES.md, you can create records whilst setting Boolean doInsert to false, this give you a list of records that you can upsert using the external ID like you would normally.
